core/citrus-base/src/test/java/org/citrusframework/util/StringUtilsTest.java

```diff
@@ -54,4 +58,152 @@ public void isEmpty_returnsFalse_forText(String str) {
     public void isEmpty_returnsFalse_forBlankText(String str) {
         assertFalse(isEmpty(str));
     }
+
+    @Test
+    public void appendSegmentToPath() {
+        assertEquals(StringUtils.appendSegmentToUrlPath("s1", "s2"), "s1/s2");
```

Comment:
```suggestion
        assertEquals(appendSegmentToUrlPath("s1", "s2"), "s1/s2");
```

`quote` is also being imported statically, but this not. is this intentional?